ci/requirements-2.7.pip

```diff
@@ -8,3 +8,4 @@ py
 PyCrypto
```

Comment:
pls make sure to add a return at the end of the line; that's the red character in the display